packages/usa-input-mask/src/usa-input-mask.stories.js

```diff
@@ -4,10 +4,20 @@ import {
   PhoneContent,
   ZipContent,
   AlphanumericContent,
+  DisabledContent,
+  AriaDisabledContent
```

Comment:
Curious why these are controlled via JSON rather than via the argType like the other components.

Reply:
This component's variants & arguments were mostly set by JSON files, so at the time I had done this to match the other variants. It wasn't until I started editing components that didn't have JSON files that I started to set the controls via argTypes instead. I can change this one to match the others now that that's been the standard for the disabled arguments!

Reply:
Quick question actually.

If I do that, beneath each of the other variant exports I'll need to define the `disabled_state` default value. Since this component has so many variants, I feel like the code will be cleaner if we leave it defined in JSON files. How do you feel about that?